omar_qaterge.js

```diff
@@ -0,0 +1,202 @@
+//1
```

Comment:
This is the homework from week2, try to include only the homework for the current week in the pull request. You can make a new branch from master and work from there.